zebra-rpc/src/methods/types/get_block_template.rs

```diff
@@ -918,7 +918,15 @@ pub fn standard_coinbase_outputs(
     let miner_reward =
         miner_reward.expect("reward calculations are valid for reasonable chain heights");
 
-    combine_coinbase_outputs(funding_streams, miner_address, miner_reward, like_zcashd)
+    let one_time_lockbox_disbursements = network.lockbox_disbursement(height);
+
+    combine_coinbase_outputs(
+        funding_streams,
+        one_time_lockbox_disbursements,
```

Comment:
Double-check that the expected size of the coinbase transaction is still correct (when calculating how many other transactions should fit into the block).